monitoring/mongodb/alerts.yaml

```diff
@@ -139,3 +139,45 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
+    expr: |
+      count_over_time(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", rs_state="3"}[1h]) > 0
```

Comment:
Do we really want to detect that only one metric was in "RECOVERING" state in the past hour? The potential problems arise when we have this state during the entire last hour (not just once), otherwise I fear that the alert will be triggered very frequently even if there is nothing wrong...

(in this case the description must be updated)

Reply:
when recovering fails  the pod restarts ( as the livenessProbe is configured ) so the average I went for initially will be below 3, I actually went for a warning if during the last hour it has been in the recovering state for at least once, but if we want to have an alert only when it's stuck in that state , we may point that in the `MongoDbNotSynced` or the alert for the primary as the fact that the node is not present might be due to the fact that it's stuck

Reply:
I think the `MongoDbRSNotSynced` alert would make more sense yes, if we cannot make this one less sensitive to recovering states (that can arise at any time if the secondary instance is a bit behind)